### DISCUSSION TITLE
Will the User have paid for Appium Test Distribution in the further?

### DISCUSSION THREAD

**Discussion by truyenaspire** [AppiumTestDistribution/AppiumTestDistribution#1170] [Q&A]
AppiumTestDistribution is a good tool for me now. I am considering putting it in my company. I want to check that it is a free tool or do you have any plans for upgrading it as a paid version. It would be wonderful if the Owner shared the long plan with us

**Comment by saikrishna321** [comment]
@truyenaspire Will be a free tool and the code will be open sourced.

**Reply by truyenaspire** [reply]
Thank so much @saikrishna321
